gitgud/__main__.py

```diff
@@ -224,26 +223,25 @@ def handle_challenges(self, args):
             print("Challenges for level \"{}\" : \n".format(level.name))
 
         
-        for index, challenge in enumerate(level.challenges):
+        for index, challenge in enumerate(level):
             print(str(index + 1) + ": " + challenge.name)
 
     def handle_load(self, args):
         self.assert_initialized()
-        if args.level_name in all_levels:
+        if args.level_name in all_levels.keys():
```

Comment:
You should just be able to check for the level name in the all_levels object, you don't need to access the keys

Reply:
You might want the `__contains__` method, but I'm not sure. You can test that